packages/commonwealth/client/scripts/controllers/chain/ethereum/aave/proposal.ts

```diff
@@ -312,7 +312,7 @@ export default class AaveProposal extends Proposal<
       console.error(
         'Failed to fetch total voting supply at proposal start block, using hardcoded value.'
       );
-      this._votingSupplyAtStart = Web3.toWei(new BN(1_000_000_000), 'ether');
+      this._votingSupplyAtStart = new BN(Web3.toWei(Number(1_000_000_000).toString(), 'ether'));
```

Comment:
👀

Reply:
Had to pass a string in and get a BN out because of library mismatches between our imported BN and the lib dependency...

Reply:
In theory this will never get used, as it's a fallback for unavailable contract data.